docs/docsite/rst/user_guide/playbook_structure.rst

```diff
@@ -0,0 +1,323 @@
+
+There are 2 ways to write tasks, 'key value pairs' or k=v and pure yaml, which mostly describes how the module/action arguments are defined.
+
+.. code-block:: YAML
+
+    - name: this is a YAML formatted task to copy a file from the controller to the target host
+      copy:
+        src: /etc/localfile
+        dest: /tmp/remotefile
+
+    - debug: msg='this is a key value  pair task'
+
+Indentation is important in a task, to separate the task keywords from the module/action options:
+
+.. code-block:: YAML
+
+    - name: this is the name keyword
+      debug:
+        msg: 'this is the msg option of the debug action'
+      when: 'this is the conditional keyword for tasks' != 'so it must always align to the task itself'
+
+For a list of keywords and the where you can place them see ...
+
+Handlers
+--------
+
+Handlers are special tasks, not on how you define them, but on where they are located, they do not execute normally but on notification.For more information on handlers see here:
+
+
+Plays
+=====
+
+Plays are a simple mapping of hosts to tasks, plays bind the actions we define to the targets we desire to apply them to.
+
+.. code-block:: YAML
+
+  - hosts: all
+
+is the simplest play, it just targets all hosts in inventory and (by default) runs M(gather_facts) on them.
+
+
+There are many ways a play can contain tasks, the simplest one is the ``tasks`` keyword.
+
+.. code-block:: YAML
+
+    - hosts: all
+      tasks:
+        - name: this is a simple task
+          debug: msg=simple task
+
+
+This keyword is just the home for a 'list of tasks', other ways a play can contain task is via the ``pre_tasks``, ``post_tasks`` and ``handlers`` keywords.
+
+Plays can ONLY appear inside a playbook, you cannot put a play inside another play nor any other object, see playbooks below.
+
+Blocks
+------
+
+Blocks are not tasks (they do look like them), but they act as 'list of tasks' inside the 'list of tasks'
+
+.. code-block:: YAML
+
+  - hosts: all
+    pre_tasks:
+     - block:
+          - name: this is a simple task
+            debug: msg=simple task
+
+While not being tasks themselves, blocks can appear anywhere a task can as they just 'wrap' tasks.
+
+For more information on blocks see here:
+
+Roles
+-----
+
+Roles also contain tasks, but mostly in a separate file, as such, they can appear in plays in several ways,
+most of the time you see them via the ``roles`` keyword:
+
+.. code-block:: YAML
+
+   - hosts: all
+     roles:
+        - rolename
+
+But they can also be included almost anywhere a task can, via the M(include_role) and M(import_role) actions.
+
+.. code-block:: YAML
+
+   - hosts: all
+     tasks:
+        - include_role: name=rolename
+     post_tasks:
+        - include_role: name=otherrole
+
+For more information on roles see here:
+
+
+Playbooks
+=========
```

Comment:
Since a new user is likely reading this doc in order to know what a "playbook" is, I think we should have a very simple playbook example as the very first topic instead of buried deep within this doc as the last point. We could then begin discussing the separate elements within the sample playbook in depth. More detailed/complex playbooks could be shown at the end, if desired.